src/main/java/de/orchound/gameoflife/model/Game.java

```diff
@@ -66,11 +69,29 @@ public void toggleCell(Vector2ic cell) {
 		if (!cellInBoardRange(cell))
 			return;
 
-		if (board.getCellStatus(cell.y(), cell.x()))
+		if (board.getCellStatus(cell.y(), cell.x())) {
 			board.killCell(cell.y(), cell.x());
-		else
+			paintMode = false;
+		}
+		else {
 			board.resurrectCell(cell.y(), cell.x());
+			paintMode = true;
+		}
+		notifyBoardDataObservers();
 
+
+	}
+
+	public void setCell(Vector2ic cell) {
```

Comment:
That's a pretty confusing / misleading name, since it suggests that we either set a cell to living or set it to some state provided as parameter.
The occurring state change of the cell depends on the "paint mode", so maybe we should call it `drawCell()`?

I still think it wouldn't be very clear to the client what this method ends up doing to the cell.
It looks like cell painting is so arbitrary, the determination of what cells to set or unset should be taken care of by another class.

`Game` is for representing and advancing the game state. It shouldn't implement the drawing logic.
You should rather add a function or two to set and unset individual cells and then extract the drawing logic.